pkg/kotsadm/objects/kotsadm_objects.go

```diff
@@ -163,11 +163,37 @@ func UpdateKotsadmDeployment(existingDeployment *appsv1.Deployment, desiredDeplo
 	existingDeployment.Spec.Template.Spec.Containers[containerIdx].VolumeMounts = desiredVolumeMounts
 	existingDeployment.Spec.Template.Spec.Containers[containerIdx].Env = desiredDeployment.Spec.Template.Spec.Containers[0].Env
 
-	existingDeployment.Spec.Template.Annotations["pre.hook.backup.velero.io/command"] = `["/scripts/backup.sh"]`
+	updateKotsadmDeploymentScriptsPath(existingDeployment)
 
 	return nil
 }
 
+func updateKotsadmDeploymentScriptsPath(existing *appsv1.Deployment) {
+	if existing.Spec.Template.Annotations != nil {
+		existing.Spec.Template.Annotations["pre.hook.backup.velero.io/command"] = `["/scripts/backup.sh"]`
+	}
```

Comment:
I think we should create the map if it is nil, but this is probably fine in practice